usr/src/uts/common/fs/zfs/sys/zap.h

```diff
@@ -88,23 +89,15 @@ extern "C" {
 
 /*
  * Specifies matching criteria for ZAP lookups.
+ * MT_CASE - Used to keep track of whether the lookup request is
+ *           case sensitive or not (i.e. Windows vs Unix.)
```

Comment:
Same as below.

First flag is about using the FS normalization settings (both unicode and case), so MT_IGNORE_FORM makes sense here, will rename.

Second flag is about honoring the case in lookup request, so it should be something like MT_PRESERVE_CASE.